packages/cli/test/simulation/multi_fork.test.ts

```diff
@@ -30,13 +42,14 @@ const env = SimulationEnvironment.initWithDefaults(
       ALTAIR_FORK_EPOCH: altairForkEpoch,
       BELLATRIX_FORK_EPOCH: bellatrixForkEpoch,
       GENESIS_DELAY: genesisSlotsDelay,
+      TERMINAL_TOTAL_DIFFICULTY: ttd,
     },
   },
   [
-    {id: "node-1", cl: CLClient.Lodestar, el: ELClient.Geth, keysCount: 32},
-    {id: "node-2", cl: CLClient.Lodestar, el: ELClient.Geth, keysCount: 32},
+    {id: "node-1", cl: CLClient.Lodestar, el: ELClient.Geth, keysCount: 32, remote: true, mining: true},
+    {id: "node-2", cl: CLClient.Lodestar, el: ELClient.Nethermind, keysCount: 32, remote: true},
```

Comment:
Just have a 1 node with remote signer, pick the second to not be also the miner

Reply:
I made the second node as remote signer. All others are not. And first node is the only miner in the set.